examples/noteprintf/noteprintf_main.c

```diff
@@ -59,7 +59,7 @@ int main(int argc, FAR char *argv[])
   short s        = 2;
   int i          = 3;
   long l         = 4;
-#ifdef CONFIG_LIBC_LONG_LONG
+#ifdef CONFIG_HAVE_LONG_LONG
```

Comment:
@xiaoxiang781216 is this still a valid change?
Should line 87 be wrapped?

Reply:
Yes, done.

Reply:
So print code still depend on HAVE, but not on LIBC?

Reply:
Yes, not only print but all code which use long long must depend on CONFIG_HAVE_LONG_LONG since the old compiler mayn't support long long type at all(long long is defined in c99).